test/integration.test.js

```diff
@@ -3,57 +3,59 @@
 const tap = require('tap')
 const test = tap.test
 const fastify = require('fastify')()
-const request = require('request')
 const plugin = require('../')
 
 fastify.register(plugin, { keys: new Set(['123456']) })
 
-fastify.get('/test1', (req, res) => {
-  res.send({ hello: 'world' })
-})
-
-fastify.get('/test2', (req, res) => {
-  res.send({ hello: 'world' })
-})
-
-fastify.get('/test3', (req, res) => {
+fastify.get('/test', (req, res) => {
   res.send({ hello: 'world' })
 })
 
 fastify.listen(0, (err) => {
```

Comment:
It should be possible to refactor this code to remove the `fastify.listen` and simply use `fastify.inject`. See https://www.fastify.io/docs/latest/Testing/

Reply:
@jsumners I get it, tried to refactor with your suggestions below, please check and confirm.